package.json

```diff
@@ -4,13 +4,13 @@
   "description": "Use this to save yourself some time when working on a webpack configuration.",
   "main": "index.js",
   "scripts": {
-    "cover": "nyc --reporter=lcov --reporter=text npm t",
+    "cover": "nyc --reporter=lcov --reporter=text ava --coverage",
```

Comment:
As I observed now (at least in my environment): the `cover` script outputs the `.nyc_output` and `coverage` folders even with the original `npm t` command; that is, `ava` without `--coverage`.
The latter does not change anything.